src/DateRangePicker.jsx

```diff
@@ -6,12 +6,12 @@ import calendar from 'calendar';
 
 import BemMixin from './utils/BemMixin';
 import CustomPropTypes from './utils/CustomPropTypes';
-import Legend from './Legend';
+import Legend from './Legend.jsx';
```

Comment:
I think I read that post. Need to investigate it again as it is a pain in all my projects :)